src/main/scala/io/sdkman/changelogs/MuleFlowDiagramsMigrations.scala

```diff
@@ -0,0 +1,38 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "027")
+class MuleFlowDiagramsMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_muleflowdiagrams_candidate",
+    author = "manikmagar"
+  )
+  def migration001(implicit db: MongoDatabase): Candidate = {
+    Candidate(
+      candidate = "muleflowdiagrams",
+      name = "Mule Flow Diagrams",
+      description =
+        "Mule application is a collection of multiple flows and sub-flows, spread across multiple configuration files. This Mule Flow Diagrams application lets you generate flow dependency graph and diagrams for your mule applications.",
+      websiteUrl = "https://github.com/manikmagar/mule-flow-diagrams/",
+      distribution = "UNIVERSAL"
+    ).insert()
+  }
+
+  @ChangeSet(
+    order = "002",
+    id = "002_add_muleflowdiagrams_0_20_0_1",
+    author = "manikmagar"
+  )
+  def migration002(implicit db: MongoDatabase) = {
```

Comment:
I suggest we leave out this migration as this will be done using the API shortly. Let's first just get the candidate added.

Reply:
Hi @marc0der , I have updated PR to remove this migration.